amgut/lib/data_access/ag_data_access.py

```diff
@@ -625,12 +628,18 @@ def AGGetBarcodeMetadataAnimal(self, barcode):
         return return_res
 
     def getAnimalParticipants(self, ag_login_id):
-        results = self._sql.execute_proc_return_cursor(
-            'ag_get_animal_participants', [ag_login_id])
-
-        return_res = [row[0] for row in results]
-        results.close()
-        return return_res
+        sql = """SELECT participant_name FROM ag_animal_survey
```

Comment:
is `ag_animal_survey` the table from the old database where we were storing responses? Why do you need to get the participant name from more than one place? It seems like you should only need to get it from one place (and if you're currently having to look in multiple places, they should be consolidated into a single place -- after all, the questions on the pet survey didn't actually change, right?)